corehq/apps/smsforms/models.py

```diff
@@ -117,6 +118,8 @@ def mark_completed(self, completed):
         self.session_is_open = False
         self.completed = completed
         self.modified_time = self.end_time = utcnow()
+        if toggles.ONE_PHONE_NUMBER_MULTIPLE_CONTACTS.enabled(self.domain):
```

Comment:
Does this need to be behind the flag? I don't expect people to be flipping this flag willy-nilly, but if someone turns it on briefly by mistake, it's be nice not to have open channels left around.

Reply:
Uh, hmm. Yeah that's a good point. I approve of that reasoning :)

Reply:
OK I thought a bit more about this, and I think right now my main concern is to have this feature fully isolated behind this flag as much as possible, so we can be as certain as possible that deploying it to production will not break any existing functionality for projects that don't turn the flag on.

I think I've convinced myself that if the toggle is not enabled, releasing or not releasing the claim on the channel has no effect. If you had a session claim that was never released because of edge cases around turning the flag on and off, and then you turn the flag on again, then that would be another story—the lingering channel claim from the finished session could prevent access for a week (i.e. until it expires). To address _that_, I've added 9506445, which has `handle_survey_actions` check for this edge case and manually release claims from closed sessions.